src/sentry/integrations/slack/webhooks/options_load.py

```diff
@@ -91,9 +90,7 @@ def post(self, request: Request) -> Response:
             .first()
         )
 
-        if not group or not features.has(
-            "organizations:slack-block-kit", group.project.organization
-        ):
+        if not group:
```

Comment:
sorry to intrude 😬 but with this removal, the `group_id` in the extra below can be replaced with `slack_request.group_id` since `group` will always be None when the logger is reached now

(i did this in #71105 and removed the flags there too but i'll just wait til this PR is merged and update my branch after so all the remaining flags can be removed at the same time 🫡 )

Reply:
thanks!